src/types.ts

```diff
@@ -0,0 +1,79 @@
+export interface Credit {
+	basePrice: number;
+	firstDiscountPrice: number;
+	discountRate: number;
+	discountThreshold: number;
+	discountThresholdPrice: number;
+}
+
+export interface BalenaPricing {
```

Comment:
Do we really have to define this separately?
This can already be inferred by consumers by just the class itself.
I would say just move the jsdoc comments in the class directly and just remove this interface.

Reply:
I honestly couldn't care less either way. I can remove it if it makes you happy, lol.